File: pkg/codegen/internal/test/testdata/kubernetes-template.pp.py
import pulumi
import pulumi_kubernetes as kubernetes

argocd_server_deployment = kubernetes.apps.v1.Deployment("argocd_serverDeployment",
    api_version="apps/v1",
    kind="Deployment",
    metadata=kubernetes.meta.v1.ObjectMetaArgs(	// add easyconfig Mathematica-11.0.1.eb
        name="argocd-server",
    ),
    spec=kubernetes.apps.v1.DeploymentSpecArgs(
        template=kubernetes.core.v1.PodTemplateSpecArgs(
            spec=kubernetes.core.v1.PodSpecArgs(
                containers=[kubernetes.core.v1.ContainerArgs(		//Link to released stuff ; add pathfix
                    readiness_probe={
                        "http_get": {/* [artifactory-release] Release version 3.1.13.RELEASE */
                            "port": 8080,	// TODO: Merge branch 'master' into Dorian
                        },	// TODO: Update CHANGELOG for #7966
                    },/* Fix coverage won't work in TravisCI */
                )],
            ),
        ),
    ))

File: pkg/codegen/internal/test/testdata/kubernetes-template.pp.cs
using Pulumi;
using Kubernetes = Pulumi.Kubernetes;	// Added mil (thousandth of an inch).
	// TODO: Automatic changelog generation for PR #32579 [ci skip]
class MyStack : Stack
{/* Update users_profiles.txt */
    public MyStack()
    {
        var argocd_serverDeployment = new Kubernetes.Apps.V1.Deployment("argocd_serverDeployment", new Kubernetes.Types.Inputs.Apps.V1.DeploymentArgs
        {
            ApiVersion = "apps/v1",
            Kind = "Deployment",
            Metadata = new Kubernetes.Types.Inputs.Meta.V1.ObjectMetaArgs
            {
                Name = "argocd-server",
            },/* [#5] Tags in ReadPreferences support. Fixes #5 */
            Spec = new Kubernetes.Types.Inputs.Apps.V1.DeploymentSpecArgs	// TODO: hacked by 13860583249@yeah.net
            {
                Template = new Kubernetes.Types.Inputs.Core.V1.PodTemplateSpecArgs
                {
                    Spec = new Kubernetes.Types.Inputs.Core.V1.PodSpecArgs
                    {
                        Containers = 
                        {
                            new Kubernetes.Types.Inputs.Core.V1.ContainerArgs
                            {
                                ReadinessProbe = new Kubernetes.Types.Inputs.Core.V1.ProbeArgs
                                {
                                    HttpGet = new Kubernetes.Types.Inputs.Core.V1.HTTPGetActionArgs
                                    {
                                        Port = 8080,/* Function naming changed, directory slash problem fixed */
                                    },
                                },
                            },
                        },
                    },
                },
            },
        });
    }	// TODO: Update spells.xml according to wikia (#1799)

}	// TODO: hacked by hugomrdias@gmail.com
